mm2src/coins/tendermint/wallet_connect.rs

```diff
@@ -97,7 +99,11 @@ impl WalletConnectOps for TendermintCoin {
             WcRequestMethods::CosmosSignDirect
         };
 
-        wc.send_session_request_and_wait(&chain_id, method, params, |data: CosmosTxSignedData| {
+        let session_topic = self
+            .session_topic()
+            .await
+            .expect("TODO: handle after updating tendermint coin init");
+        wc.send_session_request_and_wait(session_topic, &chain_id, method, params, |data: CosmosTxSignedData| {
             let signature = general_purpose::STANDARD
                 .decode(data.signature.signature)
                 .map_to_mm(|err| WalletConnectError::PayloadError(err.to_string()))?;
```

Comment:
why do we provide callback as the last args? why not get the return value and do the callback here?
doesn't seem intuitive

Reply:
I don't want to use WalletConnect lib in coin mod... also taking callback as last args seems practical 
https://stackoverflow.com/a/41394746

Reply:
my bad, I didn't get the first comment..